.github/workflows/pre-release.yml

```diff
@@ -107,3 +107,67 @@ jobs:
                 await logPublishInfo();
               }
             }
+
+  test-create-jazz-app-distribution:
+    if: contains(github.event.pull_request.labels.*.name, 'pre-release')
+    runs-on: ubuntu-latest
```

Comment:
Nice one, thanks!

Why is this located in the `pre-release` workflow?

How about running it when there is a change inside the `create-jazz-app` directory?